### DISCUSSION TITLE
请问如何方便的用xmake构建使用makefile的工程并且打包？

### DISCUSSION THREAD

**Discussion by wanzzhehe** [xmake-io/xmake#3641] [Q&A] [answered]
比方说如果是一些过去自己写的东西（也就是随便什么xmake-repo没有收录的第三方库），使用makefile构建，那么怎么快速集成到新的xmake工程中？

**Comment by waruqi** [comment, marked as answer]
自己在 xmake.lua 定义 package() 走 add_requires 本地集成。。或者 port 到 xmake.lua
